woocommerce/wc-admin
# Create ViewMoreList component

Fixes #415.

Follow-up of #493.

**Changes**
- Create a `ViewMoreList` component used in the _Orders_ table to display a list of products/coupons inside a popover when the '+X more' button is clicked.
- Update `Tag` component with a new `popoverContents` prop that makes it display a popover when clicked.
- Update the style of `Tag` component to match the style from Gutenberg form tokens.

**Screenshot**

 _Analytics_ > _Orders_:
![image](https://user-images.githubusercontent.com/3616980/46859877-4dce5800-ce0f-11e8-92bf-23ded7d8779e.png)

_Analytics_ > _Products_:
![image](https://user-images.githubusercontent.com/3616980/46795225-e5697300-cd49-11e8-8606-62b512df157b.png)

**Steps to test**

 _Analytics_ > _Orders_:

- Verify orders with two or more products/coupons display a '+X more' button with a popover.

_Analytics_ > _Products_:

- Filter by one product and verify the tag has the new style.
